configuration.nix

```diff
@@ -76,6 +76,7 @@ isMaximal: {
       csharp.enable = isMaximal;
       julia.enable = isMaximal;
       vala.enable = isMaximal;
+      nu.enable = isMaximal;
```

Comment:
I don't think this needs to be enabled in the maximal module. We've been inflating its size for languages that are rarely used for a while now, I would like to avoid that.